.github/workflows/chromatic.yml

```diff
@@ -27,11 +27,14 @@ jobs:
       - run: yarn workspace @department-of-veterans-affairs/web-components run build
       - run: yarn workspace @department-of-veterans-affairs/component-library run build
       - name: Publish to Chromatic
-        uses: chromaui/action@v1
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          # 👇 Chromatic projectToken, refer to the manage page to obtain it.
-          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
-          #onlyChanged: true
-          workingDir: packages/storybook
-          skip: "dependabot/**"
+        run: yarn chromatic storybook build --project-token=${{ secrets.CHROMATIC_PROJECT_TOKEN }} --only-changed=true
+        working-directory: packages/storybook
+        env:
+          STORYBOOK_BUILD_TIMEOUT: 10000000
+          # projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+      - name: Output build log details
+        if: always() 
+        run: cat /home/runner/work/component-library/component-library/packages/storybook/build-storybook.log
```

Comment:
Do we know if switching back to the chromatic github action works? If the file name change really does resolve the issue, we probably want to switch back to the action, since it'll ensure we're using the latest chromatic cli

Reply:
@micahchiang switched back and all is good